reference/constraints/Length.rst

```diff
@@ -42,7 +42,8 @@ and "50", you might add the following:
              *      min = 2,
              *      max = 50,
              *      minMessage = "Your first name must be at least {{ limit }} characters long",
-             *      maxMessage = "Your first name cannot be longer than {{ limit }} characters"
+             *      maxMessage = "Your first name cannot be longer than {{ limit }} characters",
+             *      allowEmptyString = true
```

Comment:
I would set this option to `false`. Allowing the empty string in this context doesn't seem to make much sense to me.

Reply:
Makes sense. I set this option to `false`